### PR TITLE
LR-161. setup angularjs

### PR DESCRIPTION
**Pull request by sergiy17**
missed Gemfile.lock
- [x] @vovka 
- [x] @anatoliiibrahimov 
- [x] @brouhaha88

### CODE REVIEW

**Comment by brouhaha88** [comment]
now it is ready !

File: Gemfile.lock [hunk 1]
```diff
@@ -257,6 +259,3 @@ DEPENDENCIES
   turbolinks (= 2.3.0)
   uglifier (= 2.5.3)
   web-console (= 2.0.0.beta3)
-
```
Comment (vovka) [inline Gemfile.lock]: why did you removed this section?

**Comment by vovka** [comment]
:ok:

**Comment by vovka** [comment]
:ok: 
May be merged when tests pass

**Pull request merged**